test.config.js

```diff
@@ -43,3 +45,7 @@ try {
 require("raf/polyfill")
 require("should")
 require("./src/lib/jade_hook")
+
+// Used by relay
+sd.METAPHYSICS_ENDPOINT = "http://localhost:5003"
+global.Promise = require("promise")
```

Comment:
Why is this needed?

Reply:
To catch the error caused by metaphysics not connecting to its fake url-- there is some conflict in jest's implementation of promise, this is a frequently recommended solution to avoid using it.

Reply:
Mind adding a comment to that issue? It seems odd that we're globally overwriting a mature (native) implementation of something with a library.

Reply:
Sure just added a link to the docs - https://jestjs.io/docs/en/troubleshooting#unresolved-promises